internal/lockservice/simpleLockService.go

```diff
@@ -146,44 +175,47 @@ func (ls *SimpleLockService) Release(sd Descriptors) error {
 
 }
 
-// CheckAcquired returns true if the file is acquired
-func (ls *SimpleLockService) CheckAcquired(sd Descriptors) bool {
+// CheckAcquired returns true if the file is Acquired.
+// It also returns the owner of the file.
+func (ls *SimpleLockService) CheckAcquired(sd Descriptors) (string, bool) {
```

Comment:
This is super confusing. Having one function called checkacquired and the other checkAcquired()

what's the difference between the two? if they both have significantly different functionality, their naming needs to be updated.

Reply:
Um no. One is a method on the client and the other on the service. Must be OK.

Reply:
This is my defence if you think thats still confusing(it shouldn't be and Im sure that's fine because they're methods of different structs):
`CheckAcquired` is checking in the LS whether the lock was acquired.
`CheckAcquire` is checking in the LC whether an `Acquire` call was made and it was successful.

Reply:
Ah okay, yup makes sense.